slsim/Sources/source_variability/variability.py

```diff
@@ -42,3 +43,29 @@ def sinusoidal_variability(t, amp, freq):
     :return: variability for the given time
     """
     return amp * np.sin(2 * np.pi * freq * t)
+
+
+def interpolate_variability(Image_series, Orig_timestamps, New_timestamps):
```

Comment:
input variables have conventions of lower_case. Capital letters are reserved for Classes

Reply:
fixed